### ISSUE TITLE
Jellyfin Search results for items with a ( - ) or ( : ) in them. Jellyfin wont wont unless exact search

### ISSUE THREAD

**Issue by Couchtater** [jellyfin/jellyfin#13146]
### This issue respects the following points:

- [x] This is a **bug**, not a question or a configuration issue; Please visit our forum or chat rooms first to troubleshoot with volunteers, before creating a report. The links can be found [here](https://jellyfin.org/contact/).
- [x] This issue is **not** already reported on [GitHub](https://github.com/jellyfin/jellyfin/issues?q=is%3Aopen+is%3Aissue) _(I've searched it)_.
- [x] I'm using an up to date version of Jellyfin Server stable, unstable or master; We generally do not support previous older versions. If possible, please update to the latest version before opening an issue.
- [x] I agree to follow Jellyfin's [Code of Conduct](https://jellyfin.org/docs/general/community-standards.html#code-of-conduct).
- [x] This report addresses only a single issue; If you encounter multiple issues, kindly create separate reports for each one.

### Description of the bug

So ive been trying out jellyfin and I noticed a huge issue. I run currently plex as my main. Want to have jellyfin for backup. However I noticed when searching for movies like Ant-Man. If I put "Ant Man" it will find 0 results, so I found out its beacuse my Arr suite correctly renames everything and ant man is Ant-Man. Is there a way to broaden the search results so Jellyfin picks up on 2 keywords inside File Name? 

### Reproduction steps

1. Upon loading into jellyfin
2. Search Ant Man - will show 0 results
3. Search Ant-Man - will show my 3 movies. 

### What is the current _bug_ behavior?

1. Search Ant Man - will show 0 results
2. Search Ant-Man - will show my 3 movies. 

### What is the expected _correct_ behavior?

1 Search Ant Man - should show 3 results
3. Search Ant-Man - should show my 3 movies. 

### Jellyfin Server version

10.10.0+

### Specify version number

 10.10.1

### Specify the build version

Release

### Environment

```markdown
- OS:unraid
- Linux Kernel: Unraid 6.12.10
- Virtualization:
- Clients:1
- Browser:Chrome or FireOs
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- GPU Model:UHD 770 iGPU
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:NAS
```

### Jellyfin logs

```shell
a
```

### FFmpeg logs

```shell
a
```

**Comment by felix920506** [comment]
Further actions towards a solution blocked by EFCore migration

**Comment by theguymadmax** [comment]
Duplicate of #1674
